Source/Core/VideoCommon/VideoBackendBase.cpp

```diff
@@ -412,6 +412,7 @@ void VideoBackendBase::ShutdownShared()
   g_shader_cache.reset();
   g_vertex_manager.reset();
   g_renderer.reset();
+  g_graphics_mod_manager.reset();
```

Comment:
Maybe move this above the texture cache?  Graphics mods can have texture references